docs/kernel-policy.md

```diff
@@ -0,0 +1,85 @@
+# Firecracker's Kernel Support Policy
+
+Being a virtual machine monitor, Firecracker represents a component in a
+larger stack, one in which it is tightly coupled with the guest and host
+kernels on which it is run.
+
+As of right now, Firecracker is being validated
+continuously on *4.14* and *5.10* host and guest kernels. While other
+versions and other kernel configs might work, they are not periodically
+validated in our test suite, and using them might result in unexpected behaviour.
+Once enabled, a kernel version is supported for a **minimum of 2 years**.
+
+We are validating the currently supported Firecracker releases as per
+[Firecracker’s release policy](../docs/RELEASE_POLICY.md). Adding support
```

Comment:
I would add before adding support... Starting with release v1.0 each major and minor release will specify the supported kernel versions. Subsequent patch releases will continue to support the same kernel versions as the minor release.

Reply:
Shouldn't this be mentioned in the release policy rather than here?

Reply:
Is a good idea. I propose to mention it there.